galaxykit/command.py

```diff
@@ -464,7 +464,7 @@ def main():
             elif args.operation == "create":
                 username, password, email, first_name, group, is_superuser, last_name = args.username, args.password. args.email, args.first_name, args.group, args.is_superuser, args.last_name
                 created, resp = users.get_or_create_user(
-                    client, username, password, None
+                    client, username, password, email, first_name, group, is_superuser, last_name, None,
```

Comment:
looks like the argument order is wrong?

https://github.com/ansible/galaxykit/blob/main/galaxykit/users.py#L8-L10

> `client, username, password, group, fname="", lname="", email="", superuser=False`

^ are accepted, so this should be 

```suggestion
                    client, username, password, group, first_name, last_name, email, is_superuser,
```

.. and there's probably no way you can pass a list of groups on the command line, so `group` should probably still be `None`

(as the ticket mentions, we have separate commands for user group membership)